benchmarks/torchbench_model.py

```diff
@@ -193,7 +193,7 @@ def get_skip_data(self):
     its lists of models into sets of models.
     """
 
-    benchmarks_dir = self._find_near_file(("benchmarks",))
+    benchmarks_dir = self._find_near_file(("pytorch/benchmarks",))
```

Comment:
```suggestion
    benchmarks_dir = self._find_near_file(("benchmarks", "pytorch/benchmarks"))
```

Reply:
Hi @ysiraichi, if you have a better fix in mind, can you create PR since you are more familiar with the code you created? I will drop this one. I am a bit concerned about adding `../../../../` in this PR.

Reply:
Sure thing.

Reply:
Sorry, just notice there is still an issue with the PR you made for the fix. I just quickly reverted it so it won't block our benchmark automation run.  I will merge this one first for the quick fix...